README.rst

```diff
@@ -11,52 +11,98 @@
 .. image:: https://travis-ci.org/Kitware/tangelo.svg?branch=master
     :target: https://travis-ci.org/Kitware/tangelo
 
+.. image:: https://readthedocs.org/projects/pip/badge/
+    :target: http://tangelo.readthedocs.org/en/latest/
+
 ============================================================
- Tangelo: A simple, quick, powerful web framework
+ Tangelo: A Web Application Platform for Python Programmers
 ============================================================
 
-http://tangelo.kitware.com
----------------------------
+Tangelo reimagines the "web application" by bringing Python into the fold:  in
+addition to serving standard components such as HTML, CSS, and JavaScript files
+in the usual way, Tangelo also manufactures serverside web services from Python
+files in your application.  These services might provide custom adapters to
+databases, launch complex jobs on a cluster to retrieve the results later,
+perform image analysis, or really anything that can be done in a Python script.
+The Python standard library is extensive, and the galaxy of third-party
+libraries even more so.  Now all the power of these libraries and modules is
+available for use in your web application as well.
```

Comment:
Maybe replace the last sentence with something like "Instead of needing to adapt your Python code to complex web frameworks and learning all about routing, app scaffolding, package APIs, etc., Tangelo adapts to your existing code, allowing seamless integration of your Python scripts." I'm sure you could make it sound a bit better :)